backend/src/main/java/quarano/sormas_integration/SynchContacts.java

```diff
@@ -0,0 +1,257 @@
+package quarano.sormas_integration;
+
+import io.vavr.Tuple2;
+import lombok.*;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+import quarano.account.DepartmentRepository;
+import quarano.core.web.MapperWrapper;
+import quarano.department.TrackedCase;
+import quarano.department.TrackedCaseRepository;
+import quarano.sormas_integration.backlog.ContactsSyncBacklogRepository;
+import quarano.sormas_integration.mapping.SormasContactDto;
+import quarano.sormas_integration.mapping.SormasContactMapper;
+import quarano.sormas_integration.mapping.SormasPersonDto;
+import quarano.sormas_integration.mapping.SormasPersonMapper;
+import quarano.sormas_integration.person.SormasContact;
+import quarano.sormas_integration.person.SormasPerson;
+import quarano.sormas_integration.report.ContactsSyncReport;
+import quarano.sormas_integration.report.ContactsSyncReportRepository;
+import quarano.sormas_integration.report.IndexSyncReport;
+import quarano.tracking.*;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+@Slf4j
+@Data
+@Setter(AccessLevel.NONE)
+@RequiredArgsConstructor
+@Component
+public class SynchContacts {
+
+    private final @NonNull MapperWrapper mapper;
+    private final @NonNull TrackedCaseRepository trackedCases;
+    private final @NonNull TrackedPersonRepository trackedPersons;
+    private final @NonNull DepartmentRepository departments;
+    private final @NonNull ContactsSyncReportRepository reports;
+    private final @NonNull ContactsSyncBacklogRepository backlog;
+
+    @org.springframework.beans.factory.annotation.Value("${quarano.sormas-integration.sormasurl:}")
```

Comment:
Please consider to use a configuration-properties class and component as it is done with "DepartmentProperties"